openmetadata-service/src/main/java/org/openmetadata/service/apps/ApplicationHandler.java

```diff
@@ -77,13 +64,9 @@ public static void runMethodFromApplication(
         | IllegalAccessException
         | InvocationTargetException e) {
       LOG.error("Exception encountered", e);
-      throw new UnhandledServerException("Exception encountered", e);
+      throw new UnhandledServerException(e.getCause().getMessage());
```

Comment:
@mohityadav766 lets not throw the message here, we should provide clear api response